### PR TITLE
Fix #875, updated SRT K band receiver feeds table

### LINKED ISSUE #875: Update the SRT K band feeds table
Due to the new receiver physical orientation, the feed table must be corrected.

### PR DESCRIPTION
**Pull request by giuseppe-carboni**
@mmurgia71 I kindly ask you to check if the offsets corresponds to the one you have used to override the old table. Thanks

### CODE REVIEW

**Review by mmurgia71 — APPROVED** [review]
I confirm that the new offsets are the correct ones and correspond to the ones I used to overwrite the old table.

**Comment by mmurgia71** [comment]
I attach the mfeed map obtained on 03JUN24 using 3C84 and 3C147 @ 18.3 GHz. Additional fine tuning of +2 degrees (clockwise) may be required.

mfeed_2deg

**Comment by mmurgia71** [comment]
Here below the result of a **software derotation** by **+2deg** and **+1.5deg** (clock-wise):

mfeed_derot

**Comment by giuseppe-carboni** [comment]
@aorlati, @sergiopoppi, we are having some doubts regarding the `relativePower` field of the feeds table.
Me and @mmurgia71 are not completely sure about the changed I made, we are not sure if the said field is an intrinsic property of each feed of the receiver or it is the result of a calculation that takes into account the rest position of each feed considering the telescope optical path.
TL;DR: I'm not sure whether the `relativePower` field should have been changed along with the offsets or it should have stayed the same for each feed.
